# Contains path(nodes and cordinates) generated by the node Path Planner.

string agent_id
RobotConfiguration robot_config
int32[] path_nodes

geometry_msgs/Point[] path_nodes_coordinates

